src/middlewares/authorization.js

```diff
@@ -0,0 +1,17 @@
+import verifyToken from '../utils/authentication';
+
+const authentication = (req, res, next) => {
+  const bearerHeader = req.headers['x-access-token'] || req.query.token || req.headers.authorization;
+  if (bearerHeader) {
+    const bearer = bearerHeader.split(' ');
+    const token = bearer[1];
+    verifyToken(req, res, token);
+    return next();
+  }
+  return res.status(401).json({
+    status: 401,
+    error: 'Unauthorized! Please sign in to perform this operation.'
+  });
```

Comment:
Remove the `Please sign in to perform this operation.`. Use a less specific error message since all protected routes apart from the sign-in route would we using this method.